src/include/stir/listmode/CListModeDataECAT8_32bit.h

```diff
@@ -80,8 +80,16 @@ class CListModeDataECAT8_32bit : public CListModeData
   typedef CListRecordECAT8_32bit CListRecordT;
   std::string listmode_filename;
   shared_ptr<InputStreamWithRecords<CListRecordT, bool> > current_lm_data_ptr;
-  shared_ptr<ProjDataInfo> proj_data_info_sptr;
+
   InterfileListmodeHeaderSiemens interfile_parser;
+  // members to store info from the interfile header.
+  // These tell us something about how the listmode is stored.
+  int axial_compression;
```

Comment:
I don't understand why we need these. In fact, I think (but am not sure) they are not used. See also below.